# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1771556.html
# Commit: 83b018946c0b
# Full Hash: 83b018946c0b78a2035e6d50c3ccd55eac3570a4
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-06-01 04:09:30
# Description:
#   Bug 1771556 - Handle backdrop capture failing clip check when sub-graph is visible r=gfx-reviewers,lsalzman
#   
#   This can happen when there is a long nested chain of backdrop-filters
#   and float inaccuracies cause the final capture primitive to be culled.
#   
# ==============================================================================

diff -r d2a56d51c65b -r 83b018946c0b gfx/tests/crashtests/1771556.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1771556.html	Tue May 31 22:55:32 2022 +0000
@@ -0,0 +1,14 @@
+<html reftest-zoom="3" class="reftest-wait">
+<head>
+  <style>
+  * {
+    scale: 0.906 52 2;
+    backdrop-filter: hue-rotate(1deg);
+    backface-visibility: hidden;
+  }
+  </style>
+</head>
+<body onload="document.documentElement.classList.remove('reftest-wait')">
+  <table>
+    <th>
+</body>